Formula/newrelic-infra-agent.rb

```diff
@@ -0,0 +1,63 @@
+class NewrelicInfraAgent < Formula
+  desc "New Relic infrastructure agent"
+  homepage "https://github.com/newrelic/infrastructure-agent"
+  url "https://github.com/newrelic/infrastructure-agent/archive/refs/tags/1.20.0.tar.gz"
+  sha256 "b9f6f744b8376b0ba92657237902d8a8e7c12cbb7f56c6f13f93b52255f31224"
+  license "Apache-2.0"
+  head "https://github.com/newrelic/infrastructure-agent.git"
+
+  depends_on "go@1.14" => :build
```

Comment:
Why can't it use go 1.16?

Reply:
This is our current tested version of go. In order to upgrade go we need to do a test for all features of the agent and it will take quite some time. We have this on the roadmap, but not a priority right now.

Reply:
Per Homebrew policy, this should use the latest version of Go possible. We should only use earlier versions when the latest one is known to be broken.